### DISCUSSION TITLE
renovate.json merge request in repositories with renovate.json

### DISCUSSION THREAD

**Discussion by Rbsn-joses** [renovatebot/renovate#9962] [Request Help]
I looked here on issues about, but I didn't find anything about it, would renovate recognize only the repositories that have renovate.json configured, instead of sending a merge request to add a standard renovate.json?

**Comment by rarkins** [comment]
Self-hosted or github app?

**Reply by Rbsn-joses** [reply]
self-hosted

**Reply by rarkins** [reply]
If you set `onboarding: false` in your bot config but leave `requireConfig` defaulting to `true` then it will skip all repos without a `renovate.json` but process those which do. Recommend you test on a small set of repos at first to confirm.

**Reply by Rbsn-joses** [reply]
its works, thank you
